python/fakes/randomStarFakes.py

```diff
@@ -2,37 +2,37 @@
 import lsst.afw.geom
 import lsst.afw.math
 import lsst.pex.config
-from lsst.pipe.tasks.fakes import FakeSourcesConfig, FakeSourcesTask
+from lsst.pipe.tasks.fakes import BaseFakeSourcesConfig, BaseFakeSourcesTask
 
-import FakeSourceLib as fsl
+import fakes.FakeSourceLib as fsl
 import numpy as np
 
-class RandomStarFakeSourcesConfig(FakeSourcesConfig):
+class RandomStarFakeSourcesConfig(BaseFakeSourcesConfig):
     nStars = lsst.pex.config.Field(dtype=int, default=1,
                                    doc="Number of stars to add")
     magnitude = lsst.pex.config.Field(dtype=float, default=20.0,
                                       doc="Magnitude of all stars to be added")
     margin = lsst.pex.config.Field(dtype=int, default=None, optional=True,
                                    doc="Size of margin at edge that should not be added")
-    seed = lsst.pex.config.Field(dtype=int, default=0,
+    seed = lsst.pex.config.Field(dtype=int, default=1,
```

Comment:
Simply because the code fails if the seed is zero with an InvalidParameterError with the current stack. As it was just a seed, I figured the value didnt matter